src/angular/hq/src/app/clients/client-details/client-details-summary/client-details-summary.component.html

```diff
@@ -0,0 +1,10 @@
+<div>
+  <h1 class="font-rajdhani font-semibold text-3xl">{{ (client | async)?.name }}</h1>
+
+  <div class="grid grid-flow-col gap-[30px] auto-cols-max pt-3">
+      <div class="text-gray-50">hourly rate: ${{(client | async)?.hourlyRate}}</div>
+      <div class="text-gray-50">email: {{(client | async)?.billingEmail}}</div>
+      <div class="text-gray-50">official name: {{(client | async)?.officialName}}</div>
+  </div>
+  <hq-error-display [errors]="apiErrors"></hq-error-display>
+</div>
```

Comment:
You can wrap this in an if statement that uses `as` to add an an alias for the async result. See https://angular.io/api/core/if#description

Something like this:
```
@if (client$ | async; as client)
{
  {{ client.name }}
}
```